MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2278,37 +2256,34 @@ protected void loadListFile(boolean allowNewPilots) throws IOException {
                 getCampaign().addUnit(entity, allowNewPilots, 0);
             }
 
-            // add any ejected pilots
-            for (Crew pilot : parser.getPilots()) {
-                if (pilot.isEjected()) {
+            // TODO : re-add any ejected pilots
```

Comment:
Why was this commented out?

Reply:
The inner code already was, so we were running a loop without processing anything.